tests/integration/records/records_datetime_fixture.py

```diff
@@ -82,7 +96,13 @@ def createDateTimeTable(self) -> None:
 """  # noqa
         else:
             raise NotImplementedError(f"Please teach me how to integration test {self.engine.name}")
-        self.engine.execute(create_tables)
+        if not self.connection:
```

Comment:
why do we have to check this? is there a problem with always creating a new connection / transaction from our engine?